services/wallet/token/uniswap.go

```diff
@@ -3799,7 +3799,7 @@ var uniswapTokens = []*Token{
 		Address:   common.HexToAddress("0x55d398326f99059fF775485246999027B3197955"),
 		Name:      "Tether USD",
 		Symbol:    "USDT",
-		Decimals:  18,
+		Decimals:  6,
 		ChainID:   56,
```

Comment:
this is for the Binance Smart Chain
https://bscscan.com/address/0x55d398326f99059ff775485246999027b3197955#readContract

18 is the right value here 
<img width="1349" alt="image" src="https://github.com/user-attachments/assets/feb1ef55-d1bb-4cb1-ac94-795d5f1dd6d5">

Reply:
The entry in the uniswap token store for USDT on Arbiscan has the correct value already
![image](https://github.com/user-attachments/assets/edf559c1-a5f8-4209-8320-9203886d7558)

In general, I'd say the Uniswap token list is reliable, we should never manually modify it.

Reply:
True, just removed this change, only updating the Arbitrum one

Reply:
Ohh I didn't check that. Thanks @dlipicar and sorry for the confusion @briansztamfater I just did the local search for USDT and saw all but that one with 6 decimals, my bad cause I haven't checked the contract.

Reply:
No problem at all!